datadog-ci-uploader/requirements.txt

```
@@ -0,0 +1,4 @@
+-r https://raw.githubusercontent.com/DataDog/datadog-agent-buildimages/nschweitzer/requirements/requirements/requests_awscli.txt
```

```suggestion
-r https://raw.githubusercontent.com/DataDog/datadog-agent-buildimages/main/requirements/requests_awscli.txt
```
Will will need to do this afterwards. We might want to pin it to a certain branch version to ensure reproducibility in datadog-agent (see how it's [referenced](https://github.com/DataDog/datadog-agent/blob/main/requirements.txt#L2) today)

---

Wouldn't it be simpler to split the addition of this file in a dedicated PR, merge it, and simply reference it from main afterward?

---

Agreed, either way works - but @chouquette's proposal seems a little bit cleaner.